doc/changes/changes_1.6.0.md

```diff
@@ -2,6 +2,10 @@
 
 ## Summary
 
+Up to version 1.65.0 ITDE used `docker_exec` to access the Docker Container, e.g. to analyze the content of logfiles or execute some shell commands. With version 8 of Exasol database the format of the Docker Containers might change so that `docker_exec` is no longer possible. Instead ITDE will then need to use SSH access.
```

Comment:
move this to a new file, I released the 1.6.0 a few minutes ago

Reply:
Done